lib/src/css_box_widget.dart

```diff
@@ -59,8 +59,24 @@ class CssBoxWidget extends StatelessWidget {
     final direction = _checkTextDirection(context, textDirection);
     final padding = style.padding?.resolve(direction);
 
+    Width? maxWidthCalculated;
+    if (style.maxWidth != null && style.width != null) {
```

Comment:
```suggestion
    final maxWidth = style.maxWidth;
    final width = style.width;
    if (maxWidth != null && width != null) {
```
this way you don't have to use the bang operator below (`style.width!`), just use `width`

Reply:
also, if it's not entirely clear what an if checks for, I really like to assign names to stuff I put inside `if`, e.g. instead of
```dart
    if (maxWidth != null && width != null) {
```
I would do
```dart
    final hasWidthConstraint = maxWidth != null && width != null;
    if(hasWidthConstraint)
```

but I'd say it's fine to use since it's pretty self-explanatory what you're trying to do now
```dart
if (maxWidth.unit == Unit.percent && width.unit == Unit.px)
```